ext_emconf.php

```diff
@@ -11,14 +11,13 @@
     'version' => '7.0.0',
     'constraints' => [
         'depends' => [
-            'typo3' => '11.5.33-12.4.99',
+            'typo3' => '12.4.0-12.4.99',
             'glossary2' => '6.0.0-6.99.99',
```

Comment:
I prefer 6.0.0-0.0.0 here to allow all upcoming glossary2 versions. On API changes we can update the min. version dep. here.

Reply:
Done ☑️